snow/consensus/snowman/consensus_test.go

```diff
@@ -1333,6 +1214,12 @@ func LastAcceptedTest(t *testing.T, factory Factory) {
 
 	require.NoError(sm.RecordPoll(context.Background(), bag.Of(block2.IDV)))
 
+	lastAcceptedID, lastAcceptedHeight = sm.LastAccepted()
+	require.Equal(block1.IDV, lastAcceptedID)
+	require.Equal(block1.HeightV, lastAcceptedHeight)
+
+	require.NoError(sm.RecordPoll(context.Background(), bag.Of(block2.IDV)))
```

Comment:
The poll for block2 finalizes block1, but is not enough to finalize block2. This adds a check to make sure the poll finalizes block1 correctly and then adds one more poll before checking that block2 finalizes correctly.